float2words/float2words.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from num2words import num2words
+from decimal import Decimal, ROUND_HALF_UP
+
+
+def float2words(number, lang, sfx1='', sfx2='', connector=', ',
+                precision=2):
+    """Convert float to words by given optional parameters.
+
+    Args:
+        number (float): float number.
+        lang (str): language code (e.g. 'en_US', 'ru_RU', etc.).
+        sfx1 (str): suffix for whole part of float number
+            (e.g. 'USD', '$', 'kg', etc.).
+        sfx2 (str): suffix for decimal part of float number.
+            (e.g. 'ct.', 'g', etc.)
+        connector (str): connector will be used between whole and
+            decimal parts expressed in words.
+        precision (int): number of digits after decimal point.
+
+    Returns:
+        number_in_words (str): float number in words.
+
+    """
+    whole_number = int(number // 1)
+    # we round float number half up by given precision
+    rounded_number = Decimal('%s' % number).quantize(
+        Decimal('.%s' % ('0' * precision)),
+        rounding=ROUND_HALF_UP
+    )
+    decimal_part = int((
+        '%.{}f'.format(precision) % rounded_number
+    ).split('.')[-1])
+    # add whitespace before suffix for better readability.
+    sfx1 = ' ' + sfx1 if sfx1 else sfx1
```

Comment:
So we add whitespace every time? What if we use `,` which is default?, So it would look like `One hundred , thirteen cents` or `One hundred, thirteen cents`?

Reply:
We add whitespace before suffix only when suffix is passed. Default suffixes are equal to '' (it means - nothing, no whitespace) and default result would look like this: 'One hundred, thirteen cents' (here default connector==', ').

Reply:
My bad, I was thinking about suffix, but looking into connector default arg:).
Though I think we can make this a bit prettier.
You can dynamically create list and then use those values to join into string, like:
```python
s1 = 'blabla'
sfx1 = ''
s2 = 'lalal'
sfx2 = 'ct.'
' '.join([x for x in [s1, sfx1, s2, sfx2] if x])
```
Then we don't even need to use this clunky `+` to connect with suffixes and then in final variable too!